### PR TITLE
AI-2490 fix build

### PR DESCRIPTION
**Pull request by mmiedviediev**
SR specific adjustments for StarSpace lib.
See also https://github.com/smartrecruiters/sr-starwrap repo

### CODE REVIEW

File: python/build.sh [hunk 1]
```diff
@@ -1,34 +1,105 @@
...
+  else
+    source $VIRTUALENVWRAPPER_SCRIPT
+    rmvirtualenv $VENV_NAME
+    mkvirtualenv $VENV_NAME
+  fi
+fi
+
+pip install -r requirements.txt
+python -m pytest -vv
+
```
Comment (zh0uquan) [inline python/build.sh]:
```suggestion
deactivate
```

Comment (zh0uquan) [reply python/build.sh]: its better to deactivate the virtual env after the tests

File: python/test/test_model.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+def test_train_model(sp):
+
+    result1 = np.array(sp.getDocVector("this\tis\ttest", "\t"))
+    result2 = np.array(sp.getDocVector("this is test", " "))
+
+    print(f"result1: ${result1}")
+    print(f"result2: ${result2}")
+
+    assert (result1 == result2).all()
```
Comment (zh0uquan) [inline python/test/test_model.py:32]: is this tested? `( a == b )` produces a bool, which has no attribute all()

File: python/test/test_model.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    sp2 = sw.starSpace(arg)
+    sp2.initFromSavedModel(os.path.join("tmp", "model"))
+    sp2.initFromTsv(os.path.join("tmp", "model.tsv"))
+
+    result1 = np.array(sp2.getDocVector("this\tis\ttest", "\t"))
+    result2 = np.array(sp2.getDocVector("this is test", " "))
+    print(f"result1: ${result1}")
+    print(f"result2: ${result2}")
+
+    assert (result1 == result2).all()
```
Comment (zh0uquan) [inline python/test/test_model.py:51]: same here

File: python/test/test_model.py (hunk 2 above)
Comment (mmiedviediev) [reply python/test/test_model.py:32]: Yes, it was. Seems like Numpy related stuff. https://www.geeksforgeeks.org/how-to-compare-two-numpy-arrays/

File: python/test/test_model.py (hunk 3 above)
Comment (mmiedviediev) [reply python/test/test_model.py:51]: please see an answer bellow

File: python/test/test_model.py (hunk 2 above)
Comment (zh0uquan) [reply python/test/test_model.py:32]: ok this is a bit confusing.

**Review by zh0uquan — APPROVED** [review]

**Pull request merged**